.github/workflows/jenkins-x/upload-binaries.sh

```diff
@@ -12,7 +12,7 @@ git config --global user.email "jenkins-x@googlegroups.com"
 
 export BRANCH=$(git rev-parse --abbrev-ref HEAD)
 export BUILDDATE=$(date)
-export REV=$(git rev-parse HEAD)
+export REV=$(git rev-parse --short origin/main 2> /dev/null || echo 'unknown')
```

Comment:
If HEAD resolved to main in L13, I dont think this fixes it. The commit actually is not in main branch, I posted the screenshot in the issue ...

Reply:
Ok, applying another way to not create the empty commit